packages/frameworks/src/frameworks.ts

```diff
@@ -584,6 +584,43 @@ export const frameworks = [
     },
     getOutputDirName: async () => '.output',
   },
+  {
+    name: 'SolidStart (v1)',
+    slug: 'solidstart-1',
+    demo: 'https://solid-start-template.vercel.app',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/solid.svg',
+    tagline: 'Simple and performant reactivity for building user interfaces.',
+    description: 'A Solid app, created with SolidStart.',
+    website: 'https://start.solidjs.com',
+    envPrefix: 'VITE_',
+    detectors: {
+      every: [
+        {
+          matchPackage: 'solid-js',
+        },
+        {
+          matchPackage: '@solidjs/start',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder:
+          '`yarn install`, `pnpm install`, `npm install`, or `bun install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `vinxi build`',
+        value: 'vinxi build',
+      },
+      devCommand: {
+        value: 'vinxi dev',
+      },
+      outputDirectory: {
+        value: '.output',
+      },
+    },
+    getOutputDirName: async () => '.output',
+  },
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** flip order

These entries are matched in order until the first match happens. If a project was going to match both for some reason, we might want v1 to be the result.

I think we should switch the order of these SolidStart entries.